File: lp1/lista1/07.py
n1 = int(input("Digite o primeiro número: "))
n2 = int(input("Digite o segundo número: "))

if n1 < n2:
    print(f"{n1} é o menor")
elif n2 < n1:
    print(f"{n2} é o menor")
else:
    print(f"Os números são iguais")

File: lp1/lista1/07.c
/*
Solicitar um número inteiro N e mostrar todos os números inteiros entre 1 e
N. Utilize as estruturas de repetição FAÇA PARA.
*/
#include <stdio.h>

int main()
{
    int num;
    printf("Digite um número inteiro: \n");
    scanf("%d", &num);
    for (int i = 1; i < num; i++)
        printf("%d\n", i);

    return 0;
}